server/src/handlers/influxdb.rs

```diff
@@ -71,6 +77,303 @@ impl From<Bytes> for WriteRequest {
 
 pub type WriteResponse = ();
 
+/// Influxql response organized in the same way with influxdb
```

Comment:
Give one example and linke
```bash
{"results":[{"statement_id":0,"series":[{"name":"mymeas",
										 "columns":["time","myfield","mytag1","mytag2"],
										 "values":[["2017-03-01T00:16:18Z",33.1,null,null],
												   ["2017-03-01T00:17:18Z",12.4,"12","14"]]}]}]}

```
- https://docs.influxdata.com/influxdb/v1.8/tools/api/#query-data-with-a-select-statement